app/src/main/java/org/fossasia/openevent/app/data/network/api/IEventService.java

```diff
@@ -14,7 +14,7 @@
 import retrofit2.http.Header;
 import retrofit2.http.POST;
 
-public interface EventService {
+public interface IEventService {
```

Comment:
Please don't change the name of interfaces we are not implementing, i.e., only the interfaces in contract package need to be changed